toss/trajectory_tools.py

```diff
@@ -0,0 +1,204 @@
+# General
+import numpy as np
+from typing import Union, Callable
+
+# For Plotting
+import pyvista as pv
+
+# For working with the mesh
+import mesh_utility
```

Comment:
This is some import shenanigans. When in doubt use [relative imports](https://realpython.com/absolute-vs-relative-python-imports/) with `from xyz import zyx` (they are better anyways because you only import what you need)

so 
`from .mesh_utility import is_outside, <whatever_else_you_need>`

and then below just

`is_outside` instead of mesh_utility.is_outside